guide/16-geoenrichment-guides/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -0,0 +1,972 @@
+{
```

Comment:
<p>Would be good to add why we are seeing 10 rows in the resulting merged df here, instead of 15 rows? since it is merging 3 zip codes with 5 providers?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>this should not occur now, I have updated the csv.</p>